### PR TITLE
Add units to master per server dashboard

### PR DESCRIPTION
**Pull request by tzach**

### CODE REVIEW

File: grafana/scylla-dash-per-server.master.json [hunk 1]
```diff
@@ -675,7 +675,7 @@
                         },
                         "yaxes": [
                             {
-                                "format": "short",
+                                "format": "wps",
```
Comment (tgrabiec) [inline grafana/scylla-dash-per-server.master.json:678]: scylla_storage_proxy_coordinator_foreground_writes presents current count of such writes, we don't take a time derivative of it, so "wps" (w per second) is not appropriate.

File: grafana/scylla-dash-per-server.master.json [hunk 2]
```diff
@@ -757,7 +757,7 @@
                         },
                         "yaxes": [
                             {
-                                "format": "short",
+                                "format": "rps",
```
Comment (tgrabiec) [inline grafana/scylla-dash-per-server.master.json:760]: same comment as for wps.

File: grafana/scylla-dash-per-server.master.json [hunk 3]
```diff
@@ -1008,7 +1008,7 @@
                         },
                         "yaxes": [
                             {
-                                "format": "short",
+                                "format": "wps",
```
Comment (tgrabiec) [inline grafana/scylla-dash-per-server.master.json:1011]: it's also a queue length, so not "wps".

File: grafana/scylla-dash-per-server.master.json [hunk 4]
```diff
@@ -1089,7 +1089,7 @@
                         },
                         "yaxes": [
                             {
-                                "format": "short",
+                                "format": "rps",
```
Comment (tgrabiec) [inline grafana/scylla-dash-per-server.master.json:1092]: same comment as for wps

File: grafana/scylla-dash-per-server.master.json [hunk 5]
```diff
@@ -4078,7 +4078,7 @@
                         },
                         "yaxes": [
                             {
-                                "format": "short",
+                                "format": "ops",
```
Comment (tgrabiec) [inline grafana/scylla-dash-per-server.master.json:4081]: This is right, but we should also fix the long standing bug in the query definition, which should take irate() (time derivative) of the metric

**Review by tgrabiec — CHANGES_REQUESTED** [review]

File: grafana/scylla-dash-per-server.master.json (hunk 5 above)
Comment (tzach) [reply grafana/scylla-dash-per-server.master.json:4081]: @tgrabiec can you please open an issue for this one?

**Comment by tzach** [comment]
Thanks @tgrabiec latest commit fix the issue you pointed to
Please check

File: grafana/scylla-dash-per-server.master.json (hunk 5 above)
Comment (tgrabiec) [reply grafana/scylla-dash-per-server.master.json:4081]: There is already an issue for this: https://github.com/scylladb/scylla-grafana-monitoring/issues/158

File: prometheus/scylla_servers.yml [hunk 6]
```diff
@@ -1,4 +1 @@
-# List Scylla end points
-
-- targets:
-  - 172.17.0.2:9180
\ No newline at end of file
+  - targets: ["172.17.0.2:9180"]
```
Comment (tgrabiec) [inline prometheus/scylla_servers.yml:1]: This commit changes scylla_servers.yml, not dashboard definition

**Review by tgrabiec — CHANGES_REQUESTED** [review]

File: prometheus/scylla_servers.yml (hunk 6 above)
Comment (tzach) [reply prometheus/scylla_servers.yml:1]: Thanks, obviously a mistke

**Comment by amnonh** [comment]
This commit is no longer applicable, the file change is no longer in use and there is a different way of specifying the units.
Please open an issue if the problem still exists.

**Pull request closed without merging**